Automattic/wp-calypso
# Mastodon: allow Social connections in staging env

Flips a feature flag so `staging` can access the Mastodon Social connection without flag URL manipulation. 

## Proposed Changes

* Enable Mastodon connections for proxied staging environment. No customer-facing impact.

## Testing Instructions

* On staging, visit Tools->Marketing->Connection. With this patch, should see Mastodon listed.

## Pre-merge Checklist
